### ISSUE TITLE
Set transaction days

### ISSUE THREAD

**Issue by centiveo** [hypnodev/laravel-nordigen#1]
As I see now it is not possible to set the total days of transactions you want to receive. It is the default (90 days), but my bank supports 720 days. Is it possible to add this?

**Comment by hypnodev** [comment]
You can set the date range using this method:
```
$user->nordigenAccount($accountId, ['transactions' => ['2021-12-01', '2022-01-30']])
```

**Comment by centiveo** [comment]
We need an end user agreement to get transactions older than 90 days? Step 3 in this Quick start: https://nordigen.com/en/account_information_documenation/integration/quickstart_guide/

**Comment by hypnodev** [comment]
Yes, we need an end user agreement for transactions older than 90 days. I'll work on this, feel free to make PR if you want
